crates/ty_python_semantic/resources/primer/memory.txt

```diff
@@ -0,0 +1,4 @@
+flake8
+sphinx
+prefect
+trio
```

Comment:
Somewhat arbitrarily chosen, open to suggestions.

Reply:
sympy might be a good addition here, I think -- we have very high memory usage on it already